multitenancy/tenant-manager-api/src/main/docker/Dockerfile.jvm

```diff
@@ -1,51 +1,11 @@
-####
-# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
-#
-# Before building the container image run:
-#
-# ./mvnw package
-#
-# Then, build the image with:
-#
-# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/managed-apicurio-registry-jvm .
-#
-# Then run the container using:
-#
-# docker run -i --rm -p 8080:8080 quarkus/managed-apicurio-registry-jvm
-#
-# If you want to include the debug port into your docker image
-# you will have to expose the debug port (default 5005) like this :  EXPOSE 8080 5050
-#
-# Then run the container using :
-#
-# docker run -i --rm -p 8080:8080 -p 5005:5005 -e JAVA_ENABLE_DEBUG="true" quarkus/managed-apicurio-registry-jvm
-#
-###
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3 
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest
 
-ARG JAVA_PACKAGE=java-11-openjdk-headless
-ARG RUN_JAVA_VERSION=1.3.8
-ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
-# Install java and the run-java script
-# Also set up permissions for user `1001`
-RUN microdnf install curl ca-certificates ${JAVA_PACKAGE} \
-    && microdnf update \
-    && microdnf clean all \
-    && mkdir /deployments \
-    && chown 1001 /deployments \
-    && chmod "g+rwX" /deployments \
-    && chown 1001:root /deployments \
-    && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
-    && chown 1001 /deployments/run-java.sh \
-    && chmod 540 /deployments/run-java.sh \
-    && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/lib/security/java.security
-
-# Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
-ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
-COPY target/lib/* /deployments/lib/
-COPY target/*-runner.jar /deployments/app.jar
+ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager" \
+    AB_ENABLED=jmx_exporter
 
 EXPOSE 8080
-USER 1001
 
-ENTRYPOINT [ "/deployments/run-java.sh" ]
+USER 185
+
+COPY target/lib/* /deployments/lib/
+COPY target/*-runner.jar /deployments/app.jar
```

Comment:
This makes sense but why is it different from what we have in the main registry app's dockerfile?  In that file it looks like this:

`ADD "$APP_URL" /deployments/`

Are we creating an uber JAR in the app but not in the tenant manager?  @carlesarnal

Reply:
No, we're building the same jar `legacy-jar` in both but we were running them differently. This won't work with the new image. It should be changed and aligned to what we have in the registry one.

Reply:
@EricWittmann The [docker module](https://github.com/Apicurio/apicurio-registry/blob/master/distro/docker/pom.xml) creates a tar.gz file that packs the runner.jar and lib folder. The ADD statement will extract the contents within the tar file in the target directory.

@carlesarnal why do you think this won;t work when we are copying the same runner.jar and lib folder in the /deployments directory?

Reply:
Sorry, I explained myself in a very deficient way. I meant that this won't work without setting the env var JAVA_APP_DIR here as well. The new base image expects that env var to be set to the java app root dir. In any case, since we're building the applications the same way, let's put the dockerfiles the same way as well.